src/libs/react-utils.ts

```diff
@@ -44,6 +44,15 @@ export const useGetter = <T>(value: T): (() => T) => {
   return () => ref.current!;
 };
 
+/**
+ * Given an initial value, returns a getter/setter pair that can be used to read and update the value.
+ * Relies on useState for the setter and useGetter for the getter.
+ * */
+export const useGetSet = <T>(initialValue: T): [() => T, (value: T) => void] => {
```

Comment:
This isn't used in this PR, but it might be needed in the future, either way it was a requested useful function from Miguel